packages/mocks/node.js

```diff
@@ -1,2 +1,15 @@
-export { mswRollupPlugin } from './rollup-plugin.js';
-export { mockPlugin } from './wds-plugin.js';
+import { setupServer } from 'msw/node';
+import { _registerMockRoutes } from './_registerMockRoutes.js';
+export { rest } from './rest.js';
```

Comment:
so users should always import from 2 locations? 🤔

```js
import { registerMockRoutes } from '@web/mocks';
import { rest } from '@web/mocks/rest.js';
```

for convenience if you already import the `registerMockRoutes` I added a reexport so you can do 
```js
import { registerMockRoutes, rest } from '@web/mocks';
```

but yeah that might be wrongly used e.g. if you only import rest from the main entrypoint 

so maybe saver to have 2 imports indeed

Reply:
> so users should always import from 2 locations? 🤔

Indeed. We need this wrapper function to be able to be forward/backward compatible, we'd like for this mocking solution to be a long term solution, so using the `rest` wrapper function we can always change the underlying mocking library if we ever intend to switch (or if MSW ever does another breaking change), while still being able to support multiple versions of mocks being used, because the wrapper function only relies on standard JS and just returns an object describing the request that was made and what it's `Response` object should be. If this (longevity/forward compatibility) is not a concern for your project, I recommend just using MSW directly.